CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 1.10
+
+### Enhancements
+
+* `RealmRecyclerViewAdapter`, a new adapter base class for `RecyclerView`s.
+
+### Bug fixes
+
+* Removed `allowBackup` and `supportsRtl` from the Android manifest file.
+
+### Credits
+
+* @thesurix for adding the `RealmRecyclerViewAdapter`.
```

Comment:
It is in the CLA, but since it wasn't public on GitHub I erred on the side of caution.
Ping @thesurix, since it is you this is about 😄